parm/config/gefs/config.efcs

```diff
@@ -6,10 +6,10 @@
 echo "BEGIN: config.efcs"
 
 # TODO: the _ENKF counterparts need to be defined in config.base
-export DO_AERO=${DO_AERO_ENKF:-"NO"}
-export DO_OCN=${DO_OCN_ENKF:-"NO"}
-export DO_ICE=${DO_ICE_ENKF:-"NO"}
-export DO_WAVE=${DO_WAVE_ENKF:-"NO"}
+# export DO_AERO=${DO_AERO_ENKF:-"NO"}
+# export DO_OCN=${DO_OCN_ENKF:-"NO"}
+# export DO_ICE=${DO_ICE_ENKF:-"NO"}
+# export DO_WAVE=${DO_WAVE_ENKF:-"NO"}
```

Comment:
Just delete these options to inherit from `config.fcst`

Reply:
Done.